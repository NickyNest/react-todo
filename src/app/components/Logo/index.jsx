import React from 'react';
import {string} from 'prop-types';
import logo from 'images/logo.svg';

const Logo = ({className, alt}) => <img src={logo} className={className} alt={alt} />;

Logo.propTypes = {
    className: string,
    alt: string
};
Logo.defaultProps = {
    className: '',
    alt: ''
};

export default Logo;
