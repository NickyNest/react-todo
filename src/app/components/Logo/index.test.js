/* eslint-disable react/jsx-filename-extension */

import React from 'react';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Logo from './index';

configure({adapter: new Adapter()});

describe('Logo', () => {
    it('Logo match expected snapshot', () => {
        expect(shallow(<Logo className='logo-class-name' alt='logo-alt-name' />)).toMatchSnapshot();
    });
});
