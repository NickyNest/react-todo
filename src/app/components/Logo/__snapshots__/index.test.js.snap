// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Logo match expected snapshot 1`] = `
<img
  alt="logo-alt-name"
  className="logo-class-name"
  src="test-file-stub"
/>
`;
